js-extensions/@jenkins-cd/subs/require-transform.js

```diff
@@ -0,0 +1,49 @@
+var transformTools = require('browserify-transform-tools');
+
+var directModuleReplacements = {
+    '@jenkins-cd/logging': 'jenkins-cd-logging:jenkins-cd-logging@any',
+    '@jenkins-cd/js-extensions': 'jenkins-cd-js-extensions:jenkins-cd-js-extensions@any',
+    '@jenkins-cd/blueocean-core-js': 'jenkins-cd-blueocean-core-js:jenkins-cd-blueocean-core-js@any',
+    '@jenkins-cd/design-language': 'jenkins-cd-design-language:jenkins-cd-design-language@any',
+    'mobx': 'mobx:mobx@any',
+    'mobx-react': 'mobx-react:mobx-react@any',
+    'react': 'react:react@any',
+    'react-dom': 'react-dom:react-dom@any',
+    'react-router': 'react-router:react-router@any',
+    'react-addons-css-transition-group': 'react-addons-css-transition-group:react-addons-css-transition-group@any',
+};
```

Comment:
@cliffmeyers these are automatically `import`ed by `js-modules` this way, basically from upstream since we know they are available and everyone should get the same versions.